src/Pages/EventManager/EventManager.js

```diff
@@ -70,6 +70,7 @@ export default function EventManager(props) {
         <Button className='create-event' onClick={toggleNewEvent}>
           New Event
         </Button>
+        <div> <br></br> </div>
```

Comment:
what is this for?

Reply:
It creates a space between the "new event" button and "click on any event below to edit or delete it"
![image](https://user-images.githubusercontent.com/55632013/97252331-f9645e00-17c6-11eb-8828-b63f9b1f7e6c.png)